Telemetry/RulesTest.cs

```diff
@@ -1,19 +1,41 @@
 // Copyright (c) Microsoft. All rights reserved.
 
+using System;
+using System.Collections.Generic;
 using System.Net;
 using Helpers;
 using Helpers.Http;
+using Helpers.Models.TelemetryRules;
+using Newtonsoft.Json;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Telemetry
 {
-    public class RulesTest
+    [Collection("Telemetry Tests")]
+    public class RulesTest : IDisposable
     {
         private readonly IHttpClient httpClient;
+        private ITestOutputHelper logger;
 
-        public RulesTest()
+        private const string DEFAULT_CHILLERS_GROUP_ID = "default_Chillers";
+        private const int SEED_DATA_RETRY_COUNT = 5;
+        private const int SEED_DATA_RETRY_MSEC = 10000;
+
+        private const string RULES_ENDPOINT_SUFFIX = "/rules";
+
+        // list of rules to delete when tests are complete
+        private List<string> disposeRulesList;
+
+        public RulesTest(ITestOutputHelper logger)
         {
             this.httpClient = new HttpClient();
+            this.logger = logger;
+
+            // Make sure device groups have been created by seed data
+            Assert.True(this.ValidChillerGroup());
```

Comment:
Does "Assert" work in constructor? If so, will it fail all test?

Reply:
The idea is to fail if valid group doesn't exists because all the rules test need a valid group.

Reply:
@ppathan I understand that. My question is, does Assert work without having "Fact" decorator on the constructor? Will it fail all tests as expected?